main.go

```diff
@@ -134,10 +109,45 @@ func main() {
 	log.Info("rqlite server stopped")
 }
 
+func closeFile(f *os.File) {
+	if err := f.Close(); err != nil {
+		log.Errorf("Unable to close file: %s", err.Error())
+		os.Exit(1)
+	}
+}
+
+func createFile(path string) *os.File {
+	usr, _ := user.Current()
+	dir := usr.HomeDir
+
+	// Check in case of paths like "/something/~/something/"
+	if path[:2] == "~/" {
+		path = strings.Replace(path, "~/", dir+"/", 1)
+	}
+
+	if err := os.MkdirAll(filepath.Dir(path), 0744); err != nil {
+		log.Errorf("Unable to create path: %s", err.Error())
+		os.Exit(1)
+	}
+
+	f, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	if err != nil && !strings.Contains(err.Error(), "is a directory") {
+		log.Errorf("Unable to open file: %s", err.Error())
+		os.Exit(1)
+	}
+
+	return f
+}
+
 func reportLaunch() {
 	json := fmt.Sprintf(`{"os": "%s", "arch": "%s", "app": "rqlite"}`, runtime.GOOS, runtime.GOARCH)
 	data := bytes.NewBufferString(json)
 	client := http.Client{Timeout: time.Duration(5 * time.Second)}
-	go client.Post("https://logs-01.loggly.com/inputs/8a0edd84-92ba-46e4-ada8-c529d0f105af/tag/reporting/",
-		"application/json", data)
+	go func() {
```

Comment:
Yep, it is, and it's usual with `defer` and `go` statements. It's pretty common that one ignores the response (and thus the error checking) when using this statements on functions that return values, and the linter in this case complains. Do you see some negative impact involved in this change?